website/content/docs/k8s/operations/faq.mdx

```diff
@@ -0,0 +1,21 @@
+---
+layout: docs
+page_title: FAQ
+sidebar_title: FAQ
+description: Frequently asked questions
+---
+
+# FAQ
+
+## Upgrades
+
+- Can I upgrade directly to a specific Helm chart version or should I upgrade one patch release at a time?
+  - It is safe to upgrade directly to a specific version. Be sure to read the release notes for all versions you're upgrading
+    through and look for any breaking changes.
+- Can I upgrade in place or should I spin up a new Kubernetes cluster?
+
+  - It is always safer to spin up a new Kubernetes cluster but that is not an
+    option for most teams. Consul supports upgrading in place.
```

Comment:
I think it'd be helpful to link to the upgrade instructions here (https://www.consul.io/docs/k8s/upgrade)?